### PR TITLE
tiup: add 4 docs mirror-mirror/merge/modify/sign.md

### PR DESCRIPTION
**Pull request by Joyinqin**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

- tiup/tiup-command-mirror-merge.md
- tiup/tiup-command-mirror-modify.md
- tiup/tiup-command-mirror-sign.md
- tiup/tiup-command-mirror.md
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/4868
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by Joyinqin** [comment]
wait for #5310 to merge to fix the dead links

**Comment by TomShawn** [comment]
> wait for #5310 to merge to fix the dead links

For this PR to be merged, you can remove the reference link to #5310 and merge this PR first. Then @CharLotteiu can merge the upstream to #5310 and add the removed links back. What do you think?

File: tiup/tiup-command-mirror-sign.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+title: tiup mirror sign
+---
+
+# tiup mirror sign
+
+The `tiup mirror sign` command is used to sign the metadata files (*.json）defined in TiUP [mirror](/tiup/tiup-mirror-reference.md). These metadata files might be stored on the local file system or remotely stored using the http protocol to provide a signature entry.
```
Comment (TomShawn) [inline tiup/tiup-command-mirror-sign.md:7]:
```suggestion
The `tiup mirror sign` command is used to sign the metadata files (*.json）defined in TiUP [mirror](/tiup/tiup-mirror-reference.md). These metadata files might be stored on the local file system or remotely stored using the HTTP protocol to provide a signature entry.
```

File: tiup/tiup-command-mirror-sign.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+## Syntax
+
+```shell
+tiup mirror sign <manifest-file> [flags]
+```
+
+`<manifest-file>` is the address of the signed file, which has two forms:
```
Comment (TomShawn) [inline tiup/tiup-command-mirror-sign.md:15]:
```suggestion
`<manifest-file>` is the address of the file to be signed, which has two forms:
```

File: tiup/tiup-command-mirror-sign.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+- Network address: start with http or https, such as `http://172.16.5.5:8080/rotate/root.json`
```
Comment (TomShawn) [inline tiup/tiup-command-mirror-sign.md:17]:
```suggestion
- Network address, which starts with HTTP or HTTPS, such as `http://172.16.5.5:8080/rotate/root.json`
```

File: tiup/tiup-command-mirror-sign.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+- Local file path: relative path or absolute path
```
Comment (TomShawn) [inline tiup/tiup-command-mirror-sign.md:18]:
```suggestion
- Local file path, which is a relative path or an absolute path
```

File: tiup/tiup-command-mirror-sign.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+If it is a network address, this address must provide the following functions:
```
Comment (TomShawn) [inline tiup/tiup-command-mirror-sign.md:20]:
```suggestion
If it is a network address, this address must provide the following features:
```

File: tiup/tiup-command-mirror-sign.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+- Supports access with `http get`, at this time the complete content of the signed file (including the signatures field) should be returned
```
Comment (TomShawn) [inline tiup/tiup-command-mirror-sign.md:22]:
```suggestion
- Supports the access via `http get` that returns the complete content of the signed file (including the `signatures` field).
```

File: tiup/tiup-command-mirror-sign.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+- Supports access with `http post`, the client adds this signature to the signatures field of the content that is returned by `http get` and POST to this address
```
Comment (TomShawn) [inline tiup/tiup-command-mirror-sign.md:23]:
```suggestion
- Supports the access via `http post`. The client adds the signature to the `signatures` field of the content that is returned by `http get` and posts to this network address.
```

File: tiup/tiup-command-mirror-sign.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+
+## Options
+
+### -k, --key
+
+- Specifies the location of the private key used for signing the `{component}.json` files.
```
Comment (TomShawn) [inline tiup/tiup-command-mirror-sign.md:29]:
```suggestion
- Specifies the location of the private key used for signing the `{component}.json` file.
```

File: tiup/tiup-command-mirror-sign.md [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+- Data type: `STRING`
+- Default: "${TIUP_HOME}/keys/private.json"
```
Comment (TomShawn) [inline tiup/tiup-command-mirror-sign.md:31]:
```suggestion
- - If this option is not specified in the command, `"${TIUP_HOME}/keys/private.json"` is used by default.
```

File: tiup/tiup-command-mirror-sign.md [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+
+### --timeout
+
+- Specifies the access timeout of the network when signing through the network. The unit is in seconds.
+- Data type: `int`
```
Comment (TomShawn) [inline tiup/tiup-command-mirror-sign.md:36]:
```suggestion
- Data type: `INT`
```

File: tiup/tiup-command-mirror-sign.md [hunk 11]
```diff
@@ -0,0 +1,47 @@
...
+
+### -k, --key
+
+- Specifies the location of the private key used for signing the `{component}.json` files.
+- Data type: `STRING`
+- Default: "${TIUP_HOME}/keys/private.json"
+
+### --timeout
+
+- Specifies the access timeout of the network when signing through the network. The unit is in seconds.
```
Comment (TomShawn) [inline tiup/tiup-command-mirror-sign.md:35]:
```suggestion
- Specifies the access timeout time for signing through the network. The unit is in seconds.
```

File: tiup/tiup-command-mirror.md [hunk 12]
```diff
@@ -0,0 +1,37 @@
+---
+title: tiup mirror
+---
+
+# tiup mirror
+
+TiUP [mirror](/tiup/tiup-mirror-reference.md) is a very important concept, and TiUP currently supports two forms of mirroring:
```
Comment (TomShawn) [inline tiup/tiup-command-mirror.md:7]:
```suggestion
In TiUP, [mirror](/tiup/tiup-mirror-reference.md) is an important concept. TiUP currently supports two forms of mirroring:
```

File: tiup/tiup-command-mirror.md [hunk 13]
```diff
@@ -0,0 +1,37 @@
...
+
+- Local mirror: the TiUP client and mirror are on the same machine, and the client accesses the mirror through the file system
```
Comment (TomShawn) [inline tiup/tiup-command-mirror.md:9]:
```suggestion
- Local mirror: the TiUP client and the mirror are on the same machine, and the client accesses the mirror through the file system.
```

File: tiup/tiup-command-mirror.md [hunk 14]
```diff
@@ -0,0 +1,37 @@
...
+- Remote mirror: the TiUP client and the mirror are not on the same machine, and the client accesses the mirror through the network
```
Comment (TomShawn) [inline tiup/tiup-command-mirror.md:10]:
```suggestion
- Remote mirror: the TiUP client and the mirror are not on the same machine, and the client accesses the mirror through network.
```

File: tiup/tiup-command-mirror.md [hunk 15]
```diff
@@ -0,0 +1,37 @@
...
+
+The `tiup mirror` command is used to manage mirrors and provides various functions such as mirror creation, component distribution, key management.
```
Comment (TomShawn) [inline tiup/tiup-command-mirror.md:12]:
```suggestion
The `tiup mirror` command is used to manage mirrors and provides ways to create mirrors, distribute components, and manage keys.
```

File: tiup/tiup-command-mirror.md [hunk 16]
```diff
@@ -0,0 +1,37 @@
...
+
+## Syntax
+
+```shell
+tiup mirror <command> [flags]
+```
+
+`<command>` stands for sub-commands. For the list of supported sub-commands, refer to the command list below.
```
Comment (TomShawn) [inline tiup/tiup-command-mirror.md:20]:
```suggestion
`<command>` stands for sub-commands. For the list of supported sub-commands, refer to the [command list](#command-list) below.
```

File: tiup/tiup-command-mirror.md [hunk 17]
```diff
@@ -0,0 +1,37 @@
...
+
+## Option
+
+None
+
+## Command list
+
+- [genkey](/tiup/tiup-command-mirror-genkey.md): generates a private key file
```
Comment (TomShawn) [inline tiup/tiup-command-mirror.md:28]:
```suggestion
- [genkey](/tiup/tiup-command-mirror-genkey.md): generates the private key file
```

File: tiup/tiup-command-mirror.md [hunk 18]
```diff
@@ -0,0 +1,37 @@
...
+- [sign](/tiup/tiup-command-mirror-sign.md): signs a specific file with a private key file
```
Comment (TomShawn) [inline tiup/tiup-command-mirror.md:29]:
```suggestion
- [sign](/tiup/tiup-command-mirror-sign.md): signs a specific file using a private key file
```

File: tiup/tiup-command-mirror.md [hunk 19]
```diff
@@ -0,0 +1,37 @@
...
+- [init](/tiup/tiup-command-mirror-init.md): initiates an empty mirror
+- [set](/tiup/tiup-command-mirror-set.md): sets the current mirror
+- [grant](/tiup/tiup-command-mirror-grant.md): grants a new component owner for the current mirror
+- [publish](/tiup/tiup-command-mirror-publish.md): publishes new components to the current mirror
+- [modify](/tiup/tiup-command-mirror-modify.md): modifies the properties of the components in the current mirror
```
Comment (TomShawn) [inline tiup/tiup-command-mirror.md:34]:
```suggestion
- [modify](/tiup/tiup-command-mirror-modify.md): modifies the attributes of the components in the current mirror
```

**Comment by Joyinqin** [comment]
@TomShawn PTAL~thx @lucklove

File: tiup/tiup-command-mirror-modify.md [hunk 20]
```diff
@@ -0,0 +1,62 @@
...
+
+> **Note:**
+>
+> This option can only be applied to the component, not to the component version.
+
+### --standalone
+
+- Controls whether the component can run standalone. This option is currently **NOT available**.
+- Data type: `BOOLEAN`
+- This option is disabled by default and its default value is `false`. To enable this option, you can add this option to the command and pass in the `true` value or no value.
```
Comment (TomShawn) [inline tiup/tiup-command-mirror-modify.md:51]:
```suggestion
- This option is disabled by default and its default value is `false`. To enable this option, you can add this option to the command, and pass the `true` value or do not pass any value.
```

File: tiup/tiup-command-mirror-modify.md [hunk 21]
```diff
@@ -0,0 +1,62 @@
...
+- After the component is marked as unavailable, you can neither see it in the result list of `tiup list` nor install the new version of the component.
+- After a component version is marked as unavailable, you can neither see it in the result list of `tiup list <component>` nor install this version.
+- Data type: `BOOLEAN`
+- This option is disabled by default and its default value is `false`. To enable this option, you can add this option to the command and pass in the `true` value or no value.
+
+### --hide
+
+- Specifies whether to hide the component. If a component is hidden, you cannot see it in the result list of `tiup list`. To see the hidden component, you can use `tiup list --all`.
+- Data type: `BOOLEAN`
+- This option is disabled by default and its default value is `false`. To enable this option, you can add this option to the command and pass in the `true` value or no value.
```
Comment (TomShawn) [inline tiup/tiup-command-mirror-modify.md:41]:
```suggestion
- This option is disabled by default and its default value is `false`. To enable this option, you can add this option to the command, and pass the `true` value or do not pass any value.
```

File: tiup/tiup-command-mirror-modify.md [hunk 22]
```diff
@@ -0,0 +1,62 @@
...
+- If this option is not specified in the command, `"${TIUP_HOME}/keys/private.json"` is used by default to sign the component information.
+
+### --yank
+
+Marks a specified component or version as unavailable.
+
+- After the component is marked as unavailable, you can neither see it in the result list of `tiup list` nor install the new version of the component.
+- After a component version is marked as unavailable, you can neither see it in the result list of `tiup list <component>` nor install this version.
+- Data type: `BOOLEAN`
+- This option is disabled by default and its default value is `false`. To enable this option, you can add this option to the command and pass in the `true` value or no value.
```
Comment (TomShawn) [inline tiup/tiup-command-mirror-modify.md:35]:
```suggestion
- This option is disabled by default and its default value is `false`. To enable this option, you can add this option to the command, and pass the `true` value or do not pass any value.
```

File: tiup/tiup-command-mirror.md [hunk 23]
```diff
@@ -0,0 +1,37 @@
...
+
+## Option
+
+None
+
+## Command list
+
+- [genkey](/tiup/tiup-command-mirror-genkey.md): generates the private key file
+- [sign](/tiup/tiup-command-mirror-sign.md): signs a specific file using a private key file
+- [init]<!-- (/tiup/tiup-command-mirror-init.md) -->: initiates an empty mirror
```
Comment (TomShawn) [inline tiup/tiup-command-mirror.md:30]:
```suggestion
- init<!-- (/tiup/tiup-command-mirror-init.md) -->: initiates an empty mirror
```

File: tiup/tiup-command-mirror.md [hunk 24]
```diff
@@ -0,0 +1,37 @@
...
+- [set](/tiup/tiup-command-mirror-set.md): sets the current mirror
+- [grant](/tiup/tiup-command-mirror-grant.md): grants a new component owner for the current mirror
+- [publish](/tiup/tiup-command-mirror-publish.md): publishes new components to the current mirror
+- [modify](/tiup/tiup-command-mirror-modify.md): modifies the attributes of the components in the current mirror
+- [rotate]<!-- (/tiup/tiup-command-mirror-rotate.md) -->: updates the root certificate in the current mirror
```
Comment (TomShawn) [inline tiup/tiup-command-mirror.md:35]:
```suggestion
- rotate<!-- (/tiup/tiup-command-mirror-rotate.md) -->: updates the root certificate in the current mirror
```

**Pull request merged**